.github/pull_request_template.md

```diff
@@ -6,3 +6,12 @@
 ## Solution
 
 - Describe the solution used to achieve the objective above.
+
+## Changelog
+
+- List a general overview of the changes made by this PR
+- Try to be concise. If more detail is needed for a particular change, consider adding it to the "Solution" section
+
+## Migration Guide
+
+- If this PR affects the public API, describe how a user might need to migrate their code to support these changes
```

Comment:
```suggestion
- If this PR is a breaking change (relative to the last release of Bevy), describe how a user might need to migrate their code to support these changes
- Simply adding new functionality is not a breaking change.
- Fixing behavior that was definitely a bug, rather than a questionable design choice is not a breaking change.
```

Reply:
How can a breaking change be relative to anything other than the last release of bevy? A breaking change to something that is only in main?

Reply:
> How can a breaking change be relative to anything other than the last release of bevy? A breaking change to something that is only in main?

That's a good point, but I think some people might see this as relative to main (especially if they're actively using main). It's probably good to specify so there's no confusion.